server/services/Mailer.js

```diff
@@ -34,21 +32,18 @@ class Mailer extends helper.Mail {
   addRecipients() {
     const personalize = new helper.Personalization();
 
-    this.recipients.forEach(recipient => {
-      personalize.addTo(recipient);
-    });
+    this.recipients.forEach(recipient => personalize.addTo(recipient));
     this.addPersonalization(personalize);
   }
 
-  async send() {
+  send() {
     const request = this.sgApi.emptyRequest({
       method: 'POST',
       path: '/v3/mail/send',
       body: this.toJSON()
     });
 
-    const response = await this.sgApi.API(request);
-    return response;
+    return this.sgApi.API(request);
```

Comment:
The function that calls this method also uses async/await...which makes no sense. Using it here means this simply returns the result of the API call, you don't `await` whatever this api call returns.

Since the function that calls it calls other async functions, it makes sense to simply remove it here.

The calling function looks like this:

```
try {
      await mailer.send();  // This makes no sense with the old code, since .send() will simply return a response, there's nothing to "await" because the .send() call already waited to return the object.
      await survey.save();
      req.user.credits -= 1;
      const user = await req.user.save();

      res.send(user);
    } catch (err) {
      res.status(422).send(err);
    }
```